project_event/models/project_task.py

```diff
@@ -205,6 +205,11 @@ class Task(models.Model):
         string='Is Created From Template',
         default=False,
     )
+    spectators = fields.Integer(
+        string='Spectators',
+        required=False,
+        readonly=False,
```

Comment:
I readonly= False, you don't need to specify it

Reply:
@eilst : It is resolved